flex-squid/src/main/java/org/sonar/flex/FlexCommentAnalyser.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * SonarQube Flex Plugin
+ * Copyright (C) 2010-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.flex;
+
+import org.sonar.squidbridge.CommentAnalyser;
```

Comment:
I really doubt about the interest of keeping this dependency. This is a very simple interface with only 2 methods.
It does not even has an internal state to keep, and both method could be static.

I think It would be even simpler to replace the usages manually in every place it is used rather than keeping this.

Reply:
The intent of this PR was only to stop using squid index, basically drop FlexAstScanner. But it's true that we can now stope extending CommentAnalyzer: b8d059d.